Makefile

```diff
@@ -780,6 +780,13 @@ ifeq ($(CI),true)
 	cp /build/* $(DIST)/binaries
 endif
 
+.PHONY: release-freebsd
+release-freebsd: | $(DIST_DIRS)
```

Comment:
I think this would need to be added to `release` as well, right?

Reply:
good catch